variables.tf

```diff
@@ -0,0 +1,34 @@
+variable "system_name" {
+  type        = string
+  description = "Name of the application of service to be used with IAM Roles Anywhere"
+}
+
+variable "system_environment" {
+  type        = string
+  description = "(optional) System Environment"

```

Comment:
You can leave it if you want to but tbh you don't need to write that it's optional